### ISSUE TITLE
"Unbound variable" in latest version

### ISSUE THREAD

**Issue by robertcoltheart** [tests-always-included/mo#67]
Running in centos using 3.0.0, I get the following error:

`/usr/local/bin/mo: line 208: moFiles[@]: unbound variable`

Command used:

`echo "test" | /usr/local/bin/mo`

**Comment by fidian** [comment]
I am unable to reproduce this error. Would you send me the output from `set +o` and `shopt -p` so I can set up my system the same as yours and possibly reproduce the problem?

**Comment by fidian** [comment]
Thanks for the extra information. I still can't replicate the issue. What version of bash do you use? What version of centos? I will set up a virtual machine and attempt to replicate the problem.

It's weird that the error message says unbound variable because line 113 assigns an empty array to it.

**Comment by robertcoltheart** [comment]
Here's a bit more for you. Its on EC2 using an Amazon2 AMI:

```
bash --version
GNU bash, version 4.2.46(2)-release (x86_64-koji-linux-gnu)
```

```
cat /etc/os-release
NAME="Amazon Linux"
VERSION="2"
ID="amzn"
ID_LIKE="centos rhel fedora"
VERSION_ID="2"
PRETTY_NAME="Amazon Linux 2"
```

**Comment by robertcoltheart** [comment]
I've gone back to the latest 2.x release for now, and that seems to work fine.

**Comment by fidian** [comment]
Thanks for working with me to solve the issue, I sincerely appreciate the help.

The bug has been fixed and I pushed out 3.0.1 with the change.
